upload/catalog/controller/account/account.php

```diff
@@ -8,6 +8,7 @@ public function index() {
 		}
 
 		$this->load->language('account/account');
+		$this->data = array_merge($this->data, $this->language->getData());
```

Comment:
How many times do you want to merge?
Easier and simplier:

> $this->data = $this->load->language('account/account');
if first,
> $this->data += $this->load->language('account/account');
if data already has assigned vars.